File: AndroidMVVMRxJavaRetrofit2/app/src/main/java/com/blackbirds/shakil/androidmvvmrxjavaretrofit2/Network/APIService.java
package com.blackbirds.shakil.androidmvvmrxjavaretrofit2.Network;

import com.blackbirds.shakil.androidmvvmrxjavaretrofit2.Model.PostModel;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface APIService {
    @GET("posts")
    Observable<List<PostModel>> getPosts();
}

File: KotlinMVVMRxJavaRetrofit2/app/src/main/java/com/blackbirds/shakil/kotlinmvvmrxjavaretrofit2/Network/APIService.kt
package com.blackbirds.shakil.kotlinmvvmrxjavaretrofit2.Network

import com.blackbirds.shakil.kotlinmvvmrxjavaretrofit2.Model.PostModel
import io.reactivex.Observable
import retrofit2.http.GET

interface APIService {
    @GET("posts")
    fun getPosts(): Observable<MutableList<PostModel>>
}